text/0079-cdk-2.0.md

```diff
@@ -0,0 +1,337 @@
+---
+feature name: cdk-2.0
+start date: 2020-3-29
+rfc pr: https://github.com/aws/aws-cdk-rfcs/pull/156
+related issue: https://github.com/aws/aws-cdk-rfcs/issues/6
+---
+
+<!-- replace the blockquoted sections with your content -->
+# Summary
+
+CDK v2.0!
+
+This RFC details the strategy for building and releasing aws-cdk 2.0, in addition to changes to tooling required to support this strategy. For details on the specific features themselves, see the corresponding RFCs.
+
+# Motivation
+
+Since the CDK was announcned as "generally available", the team has tried to limit breaking changes for users. However, a handful of features have emerged that the core team believes will make the CDK significantly simpler to use. These changes require a 2.0 release to signal to users that code changes may be required for adoption.
+
+The main change of note is to consolidate all of the aws service construct libraries into a single package. The motivation for this change is detailed in [the corresponding RFC](https://github.com/aws/aws-cdk-rfcs/issues/6).
+
+The core motivation of this proposal is to detail the plan required to release this change while continuing to support users referencing the v1 modules. In short, we want to make sure users have plenty of time to transition their code to using the new module structure and make that transition as easy as possible.
+
+# Basic Example
+
+### New Module Structure
+
+*Installing aws-cdk 2.0*
+```bash
+npm install aws-cdk aws-cdk-lib
+```
+
+*Importing modules from aws-cdk 2.0*
+```typescript
+import {
+  aws_ecs as ecs,
+  aws_lambda as lambda,
+  aws_s3 as s3,
+} from 'aws-cdk-lib';
+```
+
+*Importing experimental constructs from aws-cdk 2.0*
+```typescript
+import {aws_appsync as appsync_experimental} from 'aws-cdk-lib/experimental'
+```
+
+# Design Summary
+
+CDK 2.0 consists of the following notable changes:
+
+- [Monolithic Packaging](https://github.com/aws/aws-cdk-rfcs/issues/6) - Combine all aws construct libraries into a single package.
+- [Reset all Feature Flags](https://github.com/aws/aws-cdk-rfcs/issues/55) - Delete all defined feature flags and extraneous logic for the "old" state.
+- Remove Deprecated APIs - Delete all constructs/methods marked with `@deprecated` flag.
+- Separate Experimental Code - Extract experimental APIs into a separate entrypoint.
+
+# Detailed Design
+
+## Release Stages
+
+CDK v2.0 will be released in stages. The stages also dictate if and how the v1 modules continue to be published.
```

Comment:
Maybe provide an overview of the stages before diving into them to give readers the full picture upfront